src/app/fullscreen-button/fullscreen-button.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+
+// Code from this stackoverflow issue: https://stackoverflow.com/questions/51998594/how-to-make-google-chrome-go-full-screen-in-angular-4-application
+@Component({
+  selector: 'app-fullscreen-button',
+  templateUrl: './fullscreen-button.component.html',
+  styleUrls: ['./fullscreen-button.component.scss']
+})
+export class FullscreenButtonComponent implements OnInit {
+
+  elem;
+  fullscreen: boolean;
```

Comment:
We should probably check for if the browser is fullscreen instead of keeping an internal variable for it. The user can enter fullscreen in many different ways (F11 for example).